sdk/search/azure-search-documents/azure/search/documents/indexes/_search_indexer_client.py

```diff
@@ -493,7 +526,7 @@ def get_skillsets(self, **kwargs):
         if kwargs.get('select', None):
             kwargs['select'] = ','.join(kwargs['select'])
         result = self._client.skillsets.list(**kwargs)
-        return result.skillsets
+        return [SearchIndexerSkillset._from_generated(skillset) for skillset in result.skillsets] # pylint:disable=protected-access
```

Comment:
Is this breaking?

Reply:
Nope - the namespace for `SearchIndexerSkillset` is not in generated anymore